rhizome/lib/vm_setup.rb

```diff
@@ -37,22 +37,16 @@ def vp
     @vp ||= VmPath.new(@vm_name)
   end
 
-  def prep(unix_user, public_key, private_subnets, gua, boot_image, max_vcpus, cpu_topology, mem_gib)
+  def prep(unix_user, public_key, private_subnets, gua, ip4, local_ip4, boot_image, max_vcpus, cpu_topology, mem_gib)
     interfaces
-    routes(gua, private_subnets)
+    routes6(gua, private_subnets)
+    routes4(ip4, local_ip4) unless ip4 == "''"
```

Comment:
this is disgusting, too, the parameter translation is an issue here. Moving them via json, parsing, etc. Gotta find something else, maybe we would allow nil here.